docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -246,23 +253,30 @@ CA_CERT=$(jq -r '.ignition.security.tls.certificateAuthorities[0].source' \
 
 pushd "inventory/classes/${TENANT_ID}/"
 
-yq w -i "${CLUSTER_ID}.yml" \
-  "applications[+]" "openshift4-cloudscale"
+yq eval -i ".classes +=  \"${TENANT_ID}.openshift4\"" \
```

Comment:
Please update the requirments within that document. State the minimal `yq` version required.